src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -618,7 +618,9 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
 
   resume(_threads: number = 1) {
     if (!this.#isPaused()) {
-      console.log("Warning: startMining called when miner was already started");
```

Comment:
In other places we `console.warn()`.

should we consider extending the logger type to add some sort of log level awareness?

```javascript
export type Logger = {
  log(message?: any, ...optionalParams: any[]): void;
  info(message?: any, ...optionalParams: any[]): void;
  warn(message?: any, ...optionalParams: any[]): void;
  error(message?: any, ...optionalParams: any[]): void;
};
```

When normalizing we could default info/warn/error back to log to avoid this being a breaking change.

Reply:
I think it would be great to have a more robust logging system to do this, and I believe it's on the docket (though I couldn't find an issue for it). This PR was meant to just be a quick fix though, so I'd rather get this through. _Especially_ because I hardly consider this a "warning" - there are no consequences to calling `miner_start` when the miner isn't paused, so what are we warning them against?

Reply:
Although... it _does_ start with the word "Warning:" 🦖